nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -97,9 +103,11 @@ public static void checkInstantiationForParameterizedTypedTree(
             upperBound.getAnnotationMirrors();
         boolean hasNullableAnnotation =
             Nullness.hasNullableAnnotation(annotationMirrors.stream(), config);
+        boolean hasNullableUpperBoundLibModel =
```

Comment:
I don't think we need a new variable here.  We could just do `hasNullableAnnotation = Nullness.hasNullableAnnotation(annotationMirrors.stream(), config) || handler.onOverrideTypeParameterUpperBound(baseType.tsym.toString(), i)`

Reply:
Done